cmsplugin_articles_ai/templates/cmsplugin_articles_ai/tag.html

```diff
@@ -0,0 +1,5 @@
+<span class="label label-default">
+    <a href="{% url 'tagged_articles' tag=tag.name %}" style="color: white;">
```

Comment:
I think `<a class="label label-default" ...>` would make the inline style unnecessary, anyway.